test/true_one_to_one.jl

```diff
@@ -0,0 +1,36 @@
+include("../CaloHitContribution.jl")
+
+function test_copy()
+
+	mcp1 = MCParticle()
+    mcp1.PDG = 2212
```

Comment:
There seem to be some whitespace differences here for some of the lines (tabs vs. spaces most likely)